ochami-cli

```diff
@@ -11,21 +11,43 @@ import base64
 import time
 from http.server import HTTPServer, BaseHTTPRequestHandler
 
+from ClusterShell.NodeSet import NodeSet
```

Comment:
will you need a pip install for this? probably worth mentioning in the readme

Reply:
venv?

Reply:
Another possibility is using a [requirements.txt](https://pip.pypa.io/en/stable/reference/requirements-file-format/).

Reply:
I have a branch [here](https://github.com/synackd/ochami-cmdline/tree/requirements-txt) that updates requirements.txt with PyJWT and ClusterShell and adds installation instructions in the README.md.